docs/app-publishing.md

```diff
@@ -6,7 +6,7 @@ title: Publishing a React Native Windows App to the Microsoft Store
 ## Steps to Publish a React Native Windows App to the Microsoft Store
 These are the steps to follow if you are looking to publish a React Native Windows app as a third party to the Microsoft Store. Since React Native Windows apps are Universal Windows Platform (UWP) apps, you can also see [Publish Windows apps - UWP applications | Microsoft Docs](https://docs.microsoft.com/en-us/windows/uwp/publish/) for all kinds of documentation on the UWP app publishing process. 
 
-1. Sign-up for a developer account through the Microsoft Partner Center ([Register as an app developer – Windows app development (microsoft.com)](https://developer.microsoft.com/en-us/microsoft-store/register/)). For more guidance on the sign-up process see here: [Opening a developer account - UWP applications | Microsoft Docs](https://docs.microsoft.com/en-us/windows/uwp/publish/opening-a-developer-account).
+1. Sign-up for a developer account through the Microsoft Partner Center ([Register as an app developer – Windows app development](https://developer.microsoft.com/en-us/microsoft-store/register/)). For more guidance on the sign-up process see here: [Opening a developer account - UWP applications | Microsoft Docs](https://docs.microsoft.com/en-us/windows/uwp/publish/opening-a-developer-account).
```

Comment:
Nit: don't use the /en-us/ URLs, so that the docs show in the language the user is using (it will resolve to /en-us if their system/browser locale is en-us)

```suggestion
1. Sign up for a developer account through the Microsoft Partner Center ([Register as an app developer – Windows app development](https://developer.microsoft.com/microsoft-store/register/)). For more guidance on the sign-up process see here: [Opening a developer account - UWP applications](https://docs.microsoft.com/windows/uwp/publish/opening-a-developer-account).
```

Reply:
Will update!